config/test.exs

```diff
@@ -12,6 +12,8 @@ config :trento, Trento.Repo,
   hostname: "localhost",
   port: 5433,
   pool: Ecto.Adapters.SQL.Sandbox,
+  queue_target: 200,
+  queue_interval: 2000,
```

Comment:
lgtm, but I guess this have also some kind of impact on the time of the ci/cd run, right ?

Reply:
The tests were timing out while waiting for a db connection to become available. The additional delay is in the order of milliseconds (at most a few seconds), in my understanding.